closed/autoconf/custom-hook.m4

```diff
@@ -37,12 +37,33 @@ AC_DEFUN_ONCE([CUSTOM_EARLY_HOOK],
   AC_SUBST(OPENJ9OMR_TOPDIR)
   AC_SUBST(OPENJ9_TOPDIR)
 
+  OPENJ9_BASIC_SETUP_FUNDAMENTAL_TOOLS
   OPENJ9_PLATFORM_SETUP
   OPENJDK_VERSION_DETAILS
   OPENJ9_CONFIGURE_CUDA
+  OPENJ9_CONFIGURE_NUMA
   OPENJ9_THIRD_PARTY_REQUIREMENTS
 ])
 
+AC_DEFUN([OPENJ9_BASIC_SETUP_FUNDAMENTAL_TOOLS],
+[
+  BASIC_REQUIRE_PROGS(M4, m4)
+])
+
+AC_DEFUN_ONCE([OPENJ9_CONFIGURE_NUMA],
+[
+  if test "x$OPENJDK_TARGET_CPU_ARCH" = xx86 || "x$OPENJDK_TARGET_CPU_ARCH" = xppc; then
```

Comment:
I don't think this is what you meant: you probably want `-o` instead of `||`.

Reply:
Change made.